static/js/dist/app/settings.min.js

```diff
@@ -1 +1,247 @@
-$(document).ready(function(){$("#apiResetForm").submit(function(e){return api.reset().success(function(e){user.api_key=e.data,successFlash(e.message),$("#api_key").val(user.api_key)}).error(function(e){errorFlash(e.message)}),!1}),$("#settingsForm").submit(function(e){return $.post("/settings",$(this).serialize()).done(function(e){successFlash(e.message)}).fail(function(e){errorFlash(e.responseJSON.message)}),!1});var e=localStorage.getItem("gophish.use_map");$("#use_map").prop("checked",JSON.parse(e)),$("#use_map").on("change",function(){localStorage.setItem("gophish.use_map",JSON.stringify(this.checked))})});
\ No newline at end of file
+$(document).ready(function () {
```

Comment:
As a heads up, it looks like this is editing the `.min.js` file instead of the `.js` file.

Reply:
Thanks, shot myself in the foot there and had to come back here and grab the minified file.